test/pubVerifToken.test.ts

```diff
@@ -83,4 +83,14 @@ test.each(vectors)('PublicVerifiable-Vector-%#', async (v: Vectors) => {
     expect(uint8ToHex(tokenSer)).toBe(v.token);
 
     expect(await token.verify(publicKey)).toBe(true);
+
+    const header = token.toString();
```

Comment:
I thought of suggesting a test of a header containing multiple tokens or additional attributes but in my test these parsed nicely anyway.

Reply:
agree, more coverage is needed, e.g. parsing for multiple tokens  or a mix of valid and invalid tokens. Likely in other PR, to unblock this one.